CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ CHANGELOG
 4.3.0
```

Comment:
Should this be a major version change?

Reply:
We are just adding functionality. There are no breaking changes. A minor bump seems appropriate.